lib/vagrant-cloudstack/capabilities/rdp.rb

```diff
@@ -0,0 +1,12 @@
+module VagrantPlugins
+  module Cloudstack
+    module Cap
```

Comment:
Mostly monkey see, monkey do from other plugins and doc (http://docs.vagrantup.com/v2/plugins/guest-capabilities.html)
I assume 'Cap' module is used by Vagrant to list capabilities.